mixer/pkg/runtime/handler/env_test.go

```diff
@@ -58,16 +58,16 @@ func TestEnv(t *testing.T) {
 		wg := sync.WaitGroup{}
 		wg.Add(1)
 		e.ScheduleWork(func() {
-			if got, want := e.(env).hasStrayWorkers(), true; got != want {
-				t.Errorf("hasStrayWorkers() => %t; wanted %t", got, want)
+			if got := e.(env).hasStrayWorkers(); !got {
+				t.Errorf("hasStrayWorkers() => %t; wanted %t", got, true)
 			}
 			wg.Done()
 		})
 
 		wg.Add(1)
 		e.ScheduleDaemon(func() {
-			if got, want := e.(env).hasStrayWorkers(), true; got != want {
```

Comment:
Why is this style better than original? I certainly prefer having `want` variable.

Reply:
It leads to warnings of redundant variable usage. I improved the code, now I think it's clearly simpler and better than before. PTAL.